### DISCUSSION TITLE
Drumrolls in taiko and hold notes in mania are incorrectly gathering slider velocity information.

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#15747] [Q&A] [answered]
[EDIT: Continued in #15808]

Sorry if the title isn't the best; I can't figure out what's happening here. I can only provide two seemingly related instances of its occurrence:
1. https://osu.ppy.sh/b/1494828 At 02:32:949, there's a 0.1x velocity point with a slider that lasts until 02:35:949. In taiko and mania, this note lasts until approximately 03:03. 
![osu_2021-11-22_12-16-07](https://user-images.githubusercontent.com/74262407/142932680-dca9dfdf-d656-4802-a77c-e532bce2fabd.png)
![osu_2021-11-22_12-35-31](https://user-images.githubusercontent.com/74262407/142932742-1bd6b8d2-a470-4ed9-8536-9fa6aabfa03b.png)
![osu_2021-11-22_12-35-55](https://user-images.githubusercontent.com/74262407/142932757-1762c0be-c56e-4986-b143-10035bb3a747.png)
2. https://osu.ppy.sh/b/2294946 I entered the editor and set the first velocity point to 10x. I then saved and re-entered the editor and noticed that it had reverted to 0.5. I changed the same velocity to 3x and saved it again. Upon viewing the slightly modified beatmap, the drumroll (without visible ticks) extended through the entire song and beyond, taking a total of six minutes to complete. This should not even theoretically be possible.
![osu_2021-11-22_12-37-50](https://user-images.githubusercontent.com/74262407/142932803-c38be140-5c9f-4ca7-aa86-544028476958.png)
![osu_2021-11-22_12-39-03](https://user-images.githubusercontent.com/74262407/142932840-4f1bf713-6948-47ac-8984-c8fb4d1738f9.png)
![osu_2021-11-22_12-42-48](https://user-images.githubusercontent.com/74262407/142932855-a8e9e4c9-1c27-41c4-91a1-509b0ed8c2c0.png)
No log files provided any useful information. Version 2021.1120.0-lazer
(I would've submitted as an issue, but the option isn't available?)

**Comment by bdach** [comment, marked as answer]
these are almost certainly two separate issues.

as for the first one, the original map actually had a 0.01x difficulty control point there.

![2021-11-22-223549_582x422_scrot](https://user-images.githubusercontent.com/20418176/142938875-e57db64e-963d-4e74-bc5d-9607c0a63e90.png)

so that point is a duplicate of https://github.com/ppy/osu/discussions/14835. cannot comment on the other case yet as i haven't looked into it at this stage.
